test/check-subscriptions

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/python3
```

Comment:
Directory `./test/` contains only unit tests at this moment. Please could you move it to some subdirectory (e.g. integration_tests)?

Reply:
Yes, I was waiting for this. :-)  I'll move the files away.  This might require some changes in the Cockpit CI code, but that's for the benefit of everyone so shouldn't be an issue.

Reply:
Done.